dev/src/index.ts

```diff
@@ -243,6 +243,9 @@ const MAX_CONCURRENT_REQUESTS_PER_CLIENT = 100;
  * @property {Timestamp} lastUpdateTime The update time to enforce. If set,
  *  enforces that the document was last updated at lastUpdateTime. Fails the
  *  operation if the document was last updated at a different time.
+ * @property {boolean} exists If set, enforces that the target document must
+ * exist. This property
+ * is only valid with delete operations.
```

Comment:
This is not necessarily true. It can be used for sets with exists:false. There is also a difference in behavior for true and false and undefined.